### PR TITLE
Bugfix/2363

### PR DESCRIPTION
**Pull request by vue-kacper**
### Related issues

https://github.com/DivanteLtd/vue-storefront/issues/2363

### Short description and why it's useful

User account menu autohide on select (desktop)

### CODE REVIEW

File: src/themes/default/components/core/blocks/Header/AccountIcon.vue [hunk 1]
```diff
@@ -2,10 +2,12 @@
   <div
     class="inline-flex relative dropdown"
     data-testid="accountButton"
-    @click.self="goToAccount"
+    @click.self="goToAccount();showMenu = true;"
     @keyup.enter="goToAccount"
     tabindex="0"
     role="button"
+    @mouseover="showMenu = true"
```
Comment (patzick) [inline src/themes/default/components/core/blocks/Header/AccountIcon.vue]: isn't this solve issue?
I'm little bit worried about above syntax
```js
@click.self="goToAccount();showMenu = true;"
```
it opens modal window or changes route to account, but mouse pointer is still over, does he?

Comment (vue-kacper) [reply src/themes/default/components/core/blocks/Header/AccountIcon.vue]: This is a fallback for touch devices where we won't get mouseover event.

Adding click event we are making sure that on touch devices "User menu" will appear correctly

Comment (patzick) [reply src/themes/default/components/core/blocks/Header/AccountIcon.vue]: Okay, thanks:)

**Comment by vue-kacper** [comment]
@patzick updated :)

**Comment by patzick** [comment]
Thanks! :)

**Pull request merged**